src/rust/plugin-execution-sidecar/src/sidecar_client/generator_client.rs

```diff
@@ -1,26 +0,0 @@
-use rust_proto::{
-    client_factory::services::GeneratorClientConfig,
-    graplinc::grapl::api::plugin_sdk::generators::v1beta1::client::GeneratorServiceClient,
-    protocol::service_client::{
-        ConnectError,
-        ConnectWithConfig,
-    },
-};
-
-fn get_plugin_upstream_address(plugin_id: uuid::Uuid) -> String {
-    let upstream_addr_env_var = format!("NOMAD_UPSTREAM_ADDR_plugin-{plugin_id}");
-    let upstream_addr = std::env::var(&upstream_addr_env_var).expect(&upstream_addr_env_var);
-    let address = format!("http://{upstream_addr}");
-    address
-}
```

Comment:
I'm afraid I don't see where this behavior was replaced. Plugin upstreams cannot really be done via just-env-vars (well, they can: see the solution to https://github.com/hashicorp/nomad/issues/14813 )

Reply:
fixed